src/main/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/OverhangFixingManager.java

```diff
@@ -140,15 +141,15 @@ public OverhangFixingManager(final SAMFileHeader header,
      * @param start   the start of the split, inclusive
      * @param end     the end of the split, inclusive
      */
-    public void addSplicePosition(final String contig, final int start, final int end) {
```

Comment:
Update the doc to explain what it returns.    I assume this is just to make the testing simpler?